packages/react-ui-kit/src/Typography/ActionLink/ActionLink.tsx

```diff
@@ -0,0 +1,147 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import * as React from 'react';
+import {forwardRef} from 'react';
+
+import {CSSObject} from '@emotion/react';
+
+import {COLOR_V2} from '../../Identity/colors-v2/colors-v2';
+import {Theme} from '../../Identity/Theme';
+import {LinkProps, linkStyle, filterLinkProps} from '../Link';
+
+type VisualProps = Pick<LinkProps, 'bold' | 'color' | 'fontSize' | 'textTransform'> & {
+  disabled?: boolean;
+  children: React.ReactNode;
+};
+
+type AnchorVariant = {
+  href: string;
+} & Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'onClick' | 'children' | 'disabled'>;
```

Comment:
Can you check if 'AnchorHTMLAttributes' already have 'href'.
```suggestion
type AnchorVariant = Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'onClick' | 'children' | 'disabled'> & {
  href: string;
};
```
Following the pattern used above.